nemo/export/trt_llm/nemo_ckpt_loader/nemo_file.py

```diff
@@ -411,6 +411,7 @@ def load_nemo_model(nemo_ckpt: Union[str, Path], nemo_export_dir: Union[str, Pat
 
             nemo_model_config["mcore_gpt"] = True
             nemo_model_config["max_position_embeddings"] = nemo_model_config.get("seq_length", 4096)
+            nemo_model_config["rotary_percentage"] = nemo_model_config.get("rotary_percent")
```

Comment:
Do you think we should set the default rotary_percent which is 1.0 here as well? Like below

nemo_model_config.get("rotary_percent", 1.0)

Reply:
It would be handled here: https://github.com/NVIDIA/NeMo/blob/main/nemo/export/trt_llm/converter/model_converter.py#L188

Reply:
Yeah, I was just checking the code and was gonna say not needed.

Reply:
I did a simple test. It seems not work in this case. Just added the default value.